appengine-java8/endpoints-v2-backend/build.gradle

```diff
@@ -39,14 +41,17 @@ def projectId = 'YOUR_PROJECT_ID'
 
 apply plugin: 'java'
 apply plugin: 'war'
+// [START endpoints_plugin_apply]
 apply plugin: 'com.google.cloud.tools.endpoints-framework-server'
+// [END endpoints_plugin_apply]
 apply plugin: 'com.google.cloud.tools.appengine'
 
 dependencies {
-    // For real projects: use concrete versions here instead of the '+' to make your build consistent
-    compile 'com.google.endpoints:endpoints-framework:+'
-    compile 'com.google.endpoints:endpoints-management-control-appengine:+'
-    compile 'com.google.endpoints:endpoints-framework-auth:+'
+    compile 'com.google.endpoints:endpoints-framework:2.0.8'
```

Comment:
I'm sure there is a reason this is lonely. (no tag)  Just checking.

Reply:
Yes, the dependencies below will be used in documentation for "Adding API Management" and not starting from scratch.